apps/backend/src/services/cluster/events/courses.ts

```
@@ -0,0 +1,94 @@
+
+  for (const change of changes) {
+    switch (change.type) {
+      case "added":
+        message += `\n✅ New course: <b>${change.course_name}</b> (${change.to_classification})\n`;
+        break;
+      case "deleted":
+        message += `\n🗑️ Course removed: <b>${change.course_name}</b>\n`;
+        break;
+      case "classification_changed":
+        const fromText = change.from_classification === "inprogress" ? "in progress" : change.from_classification;
+        const toText = change.to_classification === "inprogress" ? "in progress" : change.to_classification;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)